sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -88,6 +88,21 @@ To interact with these resources, one should be familiar with the following SDK
 
 For more concepts and deeper discussion, see: [Service Bus Advanced Features](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-messaging-overview#advanced-features).
 
+### Client lifetime
+
+Each of the Service Bus client types is safe to cache and use as a singleton for the lifetime of the application, which is best practice when messages are being published or read regularly or semi-regularly. The clients are responsible for efficient resource management, working to keep resource usage low during periods of inactivity and manage health during periods of higher use. The clients are disposable and require calling `DisposeAsync` is required to ensure that network resources and other unmanaged objects are properly cleaned up.
```

Comment:
I think we may want to clarify what we mean when we say "client types". There is one "client" in Service Bus but there are lower level client-like types that allow interacting with individual entities on the namespace.

Reply:
There are both CloseAsync and DisposeAsync methods on the lower level types.

Reply:
Is it worth mentioning that closing the `ServiceBusClient` will also close all the links for the connection?

Reply:
For the processor, do we want to mention what StopProcessing does as opposed to Dispose/Close? It will stop receiving but will leave any active links open.

Reply:
Huh.... I didn't realize that you went with close and dispose.

Reply:
I refactored a bit to call out both.   Does this version read ok to you?